libraries/chain/proposal_evaluator.cpp

```diff
@@ -79,10 +135,10 @@ void_result proposal_create_evaluator::do_evaluate(const proposal_create_operati
 
    _proposed_trx.validate();
 
-   // Calling the generic hardfork visitor
+   // Calling the proposal hardfork visitor
    const fc::time_point_sec& block_time = d.head_block_time();
-   graphene::chain::impl::operation_hardfork_visitor hf_visitor(block_time);
-   hf_visitor( o );
+   proposal_operation_hardfork_visitor vtor = proposal_operation_hardfork_visitor( block_time );
+   vtor( o );
```

Comment:
Performance-wise, it's better to do this check earlier, at least before initializing `_proposed_trx`.
It should be cheaper than the authority check as well.
IMHO we can let it be the first check in this function.